### DISCUSSION TITLE
using snap notify

### DISCUSSION THREAD

**Discussion by saicharan0124** [MetaMask/snaps#1183] [Q&A] [answered]
MetaMask - RPC Error: Unauthorized to perform action. Try requesting the required permission(s) first. For more information, see: https://docs.metamask.io/guide/rpc-api.html#permissions  --> Im getting this error when im trying  to use snap notify ,I have gone through all examples i cant find the solution can anyone help me ?

**Comment by ritave** [comment, marked as answer]
Hey @saicharan0124, have you added `snap_notify: {}` permission inside the [`snap.manifest.json` file](https://docs.metamask.io/guide/snaps-development-guide.html#the-snap-manifest)?

**Reply by saicharan0124** [reply]
it worked ! Thanks a lot !!
